src/server/db_slice.cc

```diff
@@ -329,6 +329,7 @@ void DbSlice::AutoUpdater::Run() {
   }
   // TBD add logic to update iterator if needed as we can now preempt in cb
 
+  fields_.it = fields_.db_slice->db_arr_[fields_.db_ind]->Launder(fields_.it, fields_.key);
```

Comment:
Please add a TODO saying that we should remove this call once we can convers DCHECK_EQ to CHECK_EQ below.

Reply:
ah, nm  - I forgot it's for a side branch. Please do not merge

Reply:
I can add this TODO but this will not be in main branch so it will get lost anyway